internal/scale.go

```diff
@@ -16,6 +16,10 @@ func ScaleInt(c fyne.Canvas, v int) int {
 	}
 }
 
+func ScaleSize(c fyne.Canvas, s fyne.Size) fyne.Size {
```

Comment:
Correct, pixel values should be (int, int) to clearly differentiate from fyne sizes.
If this is a little abstract we could add a "PixelSize" which is int, int - but I think this might pollute the API and encourage people to think that it's usable instead of normal Size

Reply:
Ok, I will propose to introduce a PixelSize then. The idea of going with an intermediate type is to discourage people to directly modify this value. As they should be considered read only for almost all of their use and should not be tempered with.